lib/evaluators/AggregateEvaluator.ts

```diff
@@ -313,3 +312,51 @@ function extractNumericValueAndTypeOrError(term: RDF.Term): { value: number, typ
   const value = parseXSDFloat(term.value);
   return { type, value };
 }
+
+function extractValue(term: RDF.Term): {value: any, type:string}  {
+  if (term.termType !== 'Literal') {
+    throw new Error('Term is not a literal');
```

Comment:
Could you include the value and termType in the error message? (this will make debugging easier should this error occur)